modules/signing_scriptworker/templates/passwords.json.erb

```diff
@@ -39,6 +40,7 @@
         ["mac-v2-signing11.srv.releng.mdc2.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["macapp"], "signing_server"],
         ["mac-v2-signing12.srv.releng.mdc2.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["macapp"], "signing_server"],
         ["mac-v2-signing13.srv.releng.mdc2.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["macapp"], "signing_server"],
-        ["https://autograph-external.prod.autograph.services.mozaws.net", "<%= scope.function_secret(["autograph_mar_release_username"]) %>", "<%= scope.function_secret(["autograph_mar_release_password"]) %>", ["autograph_mar384", "autograph_hash_only_mar384"], "autograph"]
+        ["https://autograph-external.prod.autograph.services.mozaws.net", "<%= scope.function_secret(["autograph_mar_release_username"]) %>", "<%= scope.function_secret(["autograph_mar_release_password"]) %>", ["autograph_mar384", "autograph_hash_only_mar384"], "autograph"],
```

Comment:
I didn't even know we have the release once generated in the credentials sent from SecOps. Nice 👍